### ISSUE TITLE
More lein deps :tree conflicts

### ISSUE THREAD

**Issue by xcthulhu** [bhauman/lein-figwheel#331]
Hello again,

It looks like for the latest version of `lein-figwheel` there are *still* conflicts reported by `lein deps :tree`

```
[lein-figwheel "0.5.0-3"] -> [figwheel-sidecar "0.5.0-3"] -> [org.clojure/clojurescript "1.7.170" :exclusions [org.apache.ant/ant]] -> [org.clojure/tools.reader "0.10.0-alpha3"]
 overrides
[lein-figwheel "0.5.0-3"] -> [figwheel "0.5.0-3"] -> [org.clojure/core.async "0.2.374"] -> [org.clojure/tools.analyzer.jvm "0.6.9"] -> [org.clojure/tools.reader "1.0.0-alpha1"]

Consider using these exclusions:
[lein-figwheel "0.5.0-3" :exclusions [org.clojure/tools.reader]]
```

**Comment by bhauman** [comment]
Ugh, so this is an area of ignorance.  I will look into this and make it part of my release process.

**Comment by bhauman** [comment]
Just released 0.5.0-4 that fixes these issues.
